packages/devtools_app/lib/src/logging/html_logging_screen.dart

```diff
@@ -84,6 +119,31 @@ class HtmlLoggingScreen extends HtmlScreen {
                   controller.clear();
                 }),
               div()..flex(),
+              textFiled,
+              PButton('Search')
+                ..small()
+                ..clazz('margin-left')
+                ..click(() {
+                  final html.InputElement inputElement = textFiled.element;
+                  final String value = inputElement.value.trim();
+                  _search(value);
+                }),
+              PButton('Clear')
+                ..small()
+                ..click(() {
+                  print('test button click');
+                  final html.InputElement inputElement = textFiled.element;
+                  inputElement.value = '';
+                  _search('');
+                }),
+              div()..flex(),
+              div()..flex(),
+              ToolBarCheckbox(ToolBarCheckboxDescription(name: 'hide some log', enabled: config[filterPlatformLogTag], tag: filterPlatformLogTag)).element,
```

Comment:
this checkbox needs a better name.
For example, maybe invert when it is checked and call it 'verbose'